test/test_uops_stats.py

```diff
@@ -95,6 +95,16 @@ def test_simple_matmul(self):
     # NOTE: it's hard to assert on the memory here, all depends on caching
     assert required_mem <= mem
 
+  def test_simple_matmul_half(self):
+    if getenv("PYTHON"):
+      GlobalCounters.reset()
+      N = 128
```

Comment:
does it have to be so big? PYTHON=1 is very slow

Reply:
Changed to 16x16, should be fine for the TCs we have. Would only need to be bumped up if someone added a crazy exotic large TC like the one they have in the H100s (64x16x16).